dask/_compatibility.py

```diff
@@ -0,0 +1,19 @@
+import sys
+import warnings
+
+from importlib_metadata import entry_points as _entry_points
+from packaging.version import parse as parse_version
+
+_PY_VERSION = parse_version(".".join(map(str, sys.version_info[:3])))
+
+_EMSCRIPTEN = sys.platform == "emscripten"
```

Comment:
Should these be renamed to `PY_VERSION` and `EMSCRIPTEN` given that they are in fact intended to be imported by other modules in `dask`?

Reply:
Yeah, let's remove those leading underscores now that the entire module is marked as private